app/classes/document_query.rb

```diff
@@ -192,9 +196,10 @@ def highlight(json)
 
   def highlight_fields(json)
     json.fields do
-      json.set! "title_#{@options[:language]}", { number_of_fragments: 0 }
```

Comment:
I forgot to mention that you'll also see that I pushed a couple commits to this PR that should be uncontroversial.